### PR TITLE
Add pythreejs recipe

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: recipes/pythreejs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        - python
+        - ipywidgets >=5.1.3
+
+test:
+    imports:
+        - pythreejs
+
+about:
+    home: https://github.com/jovyan/pythreejs
+    license: BSD
```
Comment (msarahan) [inline recipes/pythreejs/meta.yaml]: 2-clause or 3?

**Comment by msarahan** [comment]
Aside from license question, LGTM.

File: recipes/pythreejs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    summary: A Python / ThreeJS bridge utilizing the Jupyter widget infrastructure.
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - jasongrout
```
Comment (jakirkham) [inline recipes/pythreejs/meta.yaml]: Again guessing you guys have talked about this. Just doing my due diligence.

Are you ok with being a maintainer here, @jasongrout?

File: recipes/pythreejs/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/pythreejs/meta.yaml]: Believe it is 3-clause, right?

File: recipes/pythreejs/meta.yaml (hunk 2 above)
Comment (jasongrout) [inline recipes/pythreejs/meta.yaml]: Yep, I'm okay. Thanks. And *thanks* @SylvainCorlay for submitting these!

File: recipes/pythreejs/meta.yaml (hunk 1 above)
Comment (jasongrout) [inline recipes/pythreejs/meta.yaml]: The standard modified BSD license: https://github.com/jovyan/pythreejs/blob/master/LICENSE

Comment (jakirkham) [inline recipes/pythreejs/meta.yaml]: Cool, if we can have that in the `meta.yaml`, preferably as `BSD 3-Clause`. Then we should be good.

**Comment by SylvainCorlay** [comment]
Fixed.

**Comment by msarahan** [comment]
Merging, thanks @SylvainCorlay and @jasongrout

**Comment by pelson** [comment]
I'm just catching up with my notifications, but just wanted to welcome both @SylvainCorlay and @jasongrout to the conda-forge world. Great to have you both on board! 👍

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**